### PR TITLE
documentation withdraw, connect, listfunds, newaddr

### PR DESCRIPTION
**Pull request by fixone**
replacement for #1406

### CODE REVIEW

File: doc/Makefile [hunk 1]
```diff
@@ -18,7 +18,11 @@ MANPAGES := doc/lightning-cli.1 \
 	doc/lightning-sendpay.7 \
 	doc/lightning-waitinvoice.7 \
 	doc/lightning-waitanyinvoice.7 \
-	doc/lightning-waitsendpay.7
+	doc/lightning-waitsendpay.7 \
+	doc/lightning-newaddr.7 \
```
Comment (ZmnSCPxj) [inline doc/Makefile]: Existing list is sorted, please sort your inserted files also

File: doc/lightning-autocleaninvoice.7 [hunk 2]
```diff
@@ -2,12 +2,12 @@
 .\"     Title: lightning-autocleaninvoice
 .\"    Author: [see the "AUTHOR" section]
 .\" Generator: DocBook XSL Stylesheets v1.79.1 <http://docbook.sf.net/>
-.\"      Date: 04/16/2018
+.\"      Date: 04/26/2018
```
Comment (ZmnSCPxj) [inline doc/lightning-autocleaninvoice.7]: Unnecessary change?  Only date was changed.

**Comment by fixone** [comment]
changed order, removed autocleaninvoice from commit

**Comment by ZmnSCPxj** [comment]
ACK 621b68b62c796cce19fda01c174c9ff8584e7715
